src/shared/components/navbar/sidebar/Sidebar.tsx

```diff
@@ -20,6 +22,12 @@ const Sidebar = memo(
         {IMPORTANT_LINKS.map(link => (
           <NavLink key={link.to} activeClassName={csx.activeLink} {...link} />
         ))}
+        <Guard.Unprotected>
```

Comment:
This should be done via simple filter function - don't change implementation. Check for reference getLinksByAuthState in current master branch.

Reply:
done